store/storetest/user_store.go

```diff
@@ -322,6 +323,26 @@ func testUserStoreGet(t *testing.T, ss store.Store) {
 	})
 }
 
+func testUserStoreGetNotifyProps(t *testing.T, ss store.Store) {
+	u1 := &model.User{
+		Email: MakeEmail(),
+	}
+	_, err := ss.User().Save(u1)
+	require.NoError(t, err)
+	defer func() { require.NoError(t, ss.User().PermanentDelete(u1.Id)) }()
+
+	t.Run("fetch empty id", func(t *testing.T) {
+		_, err := ss.User().GetNotifyProps("")
+		require.Error(t, err)
+	})
+
+	t.Run("fetch user 1", func(t *testing.T) {
+		actual, err := ss.User().GetNotifyProps(u1.Id)
+		require.NoError(t, err)
+		require.Equal(t, u1, actual)
```

Comment:
You are trying to compare a `*model.User` to a `map[string]string` instance here. Hence it's failing.

Reply:
gentle ping about this comment.